beamformer.c

```diff
@@ -682,9 +699,16 @@ do_beamformer(BeamformerCtx *ctx, Arena arena)
 		v2 mouse = { .rl = GetMousePosition() };
 		if (CheckCollisionPointRec(mouse.rl, vr.rl)) {
 			f32 delta = GetMouseWheelMove();
-			ctx->fsctx.db += delta;
-			CLAMP(ctx->fsctx.db, -120, 0);
-			if (delta) ctx->flags |= GEN_MIPMAPS;
+			ctx->params->raw.off_axis_pos += delta * 1e-3;
+			if (delta)
+			{
+				ctx->flags |= DO_COMPUTE;
+				ctx->params->upload = 1;
+			}
+
+			//ctx->fsctx.db += delta;
+			//CLAMP(ctx->fsctx.db, -120, 0);
+			//if (delta) ctx->flags |= GEN_MIPMAPS;
```

Comment:
This should be left out for now as well. Also you will still need to set `GEN_MIPMAPS` otherwise the image of the new plane will be filtered incorrectly.

Reply:
Scratch that, I forgot it gets set after `do_compute_shader` is run.